subprojects/matrix-benchmarking-workloads/ansible-llm/models/calculated.py

```diff
@@ -0,0 +1,14 @@
+from matrix_benchmarking import models as matbench_models
+
+class StatusCodeDistribution(matbench_models.ExclusiveModel):
+    OK: int
+    # Don't know what other status codes there are
```

Comment:
```suggestion
class StatusCodeData(matbench_models.ExclusiveModel):
    status: str
    count: int
```

We may want to keep this more general, and then use a List in the AnsibleLLMData class, similar to LatencyDistribution.  There are many potential error statuses and I'm not sure if there is a way to enumerate all of them in maintainable way, in case e.g. the runtime or modelmesh adds new error codes / messages

Reply:
Hmmm, well we will need to do some conversion to turn the .json output from the load tester into this format anyway, so we can likely make this conversion as part of that.

Reply:
We can still do the format below, just might not be as nice to deal with (and not use the code committed above)
```
{
   "OK": 1024,
   "err1": 2,
   ...
}
```